microsite/blog/2022-09-08-fyi-plugin-analytics-api.md

```diff
@@ -0,0 +1,88 @@
+---
+# prettier-ignore
+title: FYI 📣 The Plugin Analytics API
+author: Eric Peterson, Spotify
+authorURL: https://github.com/iamEAP
+authorImageURL: https://avatars.githubusercontent.com/u/3496491?v=4
+---
+
+**TL;DR** If you didn't know, now you know: the Backstage plugin analytics API is here to help you understand how developers in your organization are using Backstage.
+
```

Comment:
This needs a fold (`<!--truncate-->`) somewhere, and prefereably some art above that as well. Otherwise it'll take up the full space.